index.html

```diff
@@ -107,38 +158,17 @@ <h2>Multi-Post Stories</h2>
             <button type="button" class="btn btn-see">See Project</button>
           </div>
         </div>
-        <div class="grid-2">
-          <div class="card">
-            <div class="block">
-              <h2 class="h-text-white h-text-size">
-                Profesional Art Printing Data
-              </h2>
-              <p class="p-text-white">
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required. has been the industry's standard
-              </p>
-              <ul class="tags2">
-                <li>
-                  <span class="tag-border tags-color-white html">HTML</span>
-                </li>
-                <li>
-                  <span class="tag-border tags-color-white bootstrap"
-                    >Bootstrap</span
-                  >
-                </li>
-                <li>
-                  <span class="tag-border tags-color-white ruby">Ruby</span>
-                </li>
-              </ul>
-            </div>
+        <div class="grid-2 display-grid">
+          <div class="card one">
             <div class="btn-group">
               <button type="button" class="btn-2">See Project</button>
             </div>
           </div>
```

Comment:
- [ ] Please notice that the previous reviewer mentioned that you should move the "See Project" button of this card specifically for this card only. 

Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/155845347-13c4227b-3f2d-4491-ac50-8fee4757e608.png)|![image](https://user-images.githubusercontent.com/18730106/155845360-d674b41f-b148-4295-bf29-a47a1e70458a.png)